liesel/model/nodes.py

```diff
@@ -631,7 +663,28 @@ def value(self) -> float:
 
 
 class Var:
-    """A variable wrapping a value and a distribution node."""
+    """
+    A variable of a statistical model, typically with a probability distribution.
+
+    A variable in Liesel is typically a random variable, e.g. an observed or
+    latent variable with a probability distribution or a model parameter with a
+    prior distribution. However, other quantities can also be declared as variables,
+    e.g. fixed data like hyperparameters or design matrices, or quantities that are
+    computed from other nodes, e.g. structured additive predictors in semi-parametric
+    regression models.
+
+    When a :class:`.Node` does not have an associated probability distribution, it is
+    not strictly necessary to declare it as a variable. There is no hard and fast rule
+    when such a node should be declared as a variable and when not. The advantages of
+    declaring a node as a variable are: (1) easier access via the :attr:`.Model.vars`
+    attribute, and (2) more explicit visualization with the :func:`.plot_vars`
+    function.
```

Comment:
I feel like this makes it sound a bit more liberal than it is. The `Node`  child classes like `Dist`, `Data`, `Calc` seem to have rather narrow purposes, and the `Node` cannot be used directly.